screens/auth/AuthScreen.js

```diff
@@ -191,18 +181,13 @@ class AuthScreen extends Component {
             );
         }
 
-        // FIXME: convert this feature to react navigation 5 -- commenting code for now
-        // this.focusListner = this.props.navigation.addListener(
-        //     'didFocus',
-        //     () => {
-        //         this.props.loginOrSignupReset();
-        //     }
-        // );
+        this.focusListner = this.props.navigation.addListener('focus', () => {
+            this.props.loginOrSignupReset();
+        });
```

Comment:
Do navigation listeners need an unsubscribe method?

Reply:
What does this even do?

Reply:
Why do we need to reset loginOrSignup? Comment needed here

Reply:
yes navigation listners need to be unsubscribed on onUnmount

loginOrSignup function was already here I just changed navigation listner code.. migrated it from react navigation v4 to v5..
Not looked into why loginOrSignup fn was implemented..